### ISSUE TITLE
Off-by-one-error in tag initialization

### ISSUE THREAD

**Issue by jasom** [blakemcbride/eclipse-lisp#1]
This line loops from 1 to 8; the array it is initializing is indexed from 0-7; The ommision of 0 is probably correct, as all zeros is the data tag.  I'm not sure if the bug is the inclusion of 8 in the loop, or the size of the cl_POINTER_TAGGED_FOO arrays should be 1 larger, so I didn't make a PR.

https://github.com/blakemcbride/eclipse-lisp/blob/master/c/interface.c#L3609

**Comment by blakemcbride** [comment]
Thank you very much for the report.  As mentioned in ISSUES.txt, the system
does have some problems that shouldn't be too bad for someone with a bit of
time.

**Comment by blakemcbride** [comment]
Good catch!  I just don't have time enough to figure it out and fix it.  A PR would be great!

**Comment by blakemcbride** [comment]
This has been checked.  The existing code is correct.
